dev-vcs/git/git-2.45.2.recipe

```diff
@@ -133,7 +133,7 @@ REQUIRES_scalar="
 	"
 
 # The `subversion` package includes perl bindings that this tool needs.
-# Note: `subversion` still built with gcc2 on 32 bits (matching perl there, at least).
+# Note: `subversion` still built with gcc2 on 32 bits.
```

Comment:
This needs to be checked if this still works (perl is now using gcc 13 on x86).

Reply:
subversion failed in the INSTALL part for the bindings libsvn_swig_perl on 32bit

Reply:
Thanks. I suspected that. I think we should switch subversion to secondaryArch then.

Reply:
I don't know who invented "portPackageLinksDir", but I'm never getting this right ;)
Seems cm:apr_1_config$secondaryArchSuffix should do it (still checking)

Reply:
Test result on 64bit are the same as in the subversion recipe (no supprise as there isn't anything really changed), but still. :)

Reply:
@jmairboeck PR for subversion: https://github.com/haikuports/haikuports/pull/10628
Maybe you could check it out? (testcase on 32bit still running)

Reply:
I won't have much time to test anything before the weekend, probably.

Reply:
Test results from 32bit: same as on 64bit, so I guess the PR should be good to go. :)